tests/dummy/app/index.html

```diff
@@ -24,7 +24,8 @@
     <script src="{{rootURL}}assets/dummy.js"></script>
 
     {{content-for "body-footer"}}
-    <!-- Place this tag right after the last button or just before your close body tag. -->
-    <script async defer id="github-bjs" src="https://buttons.github.io/buttons.js"></script>
+
+    {{!-- Place this tag in your head or just before your close body tag. --}}
```

Comment:
We need to use html comments here

```suggestion
    <!-- Place this tag in your head or just before your close body tag. -->
```

Reply:
oh it doesn't let me commit the suggestion for some reason 🤔

Reply:
probably due to coming from the @linc-technologies upstream? 🤔
We have some default org perms configured :)